kms/src/main/java/io/kroxylicious/kms/service/Ser.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.service;
```

Comment:
While the KMS is the only current usage of this interface I'm not convinced it is the right home. A `ByteBuffer` sympathetic Serde feels sufficiently useful throughout the kroxylicious runtime.

Having said that I'm entirely happy to leave it here until we have a 2nd consumer for it.

Reply:
I agree it's not entirely right it living here, but I also agree to keep it here for now.